### ISSUE TITLE
How do you play only from a single album?

### ISSUE THREAD

**Issue by rubenwardy** [gnumdk/lollypop#1049]
* Lollypop version: 0.9.231
* Gtk Version: 2.24.30, 3.20.9 according to dpkg
* Linux distribution: Xbuntu 16.10 (xfce 4.12)

How can I make it only play tracks from a certain album? When I click a track in an album, I expected it to only play tracks in that album afterwards, however it seems to be sampling from the whole library. Is there a config option for this?

Also, the shuffle always seems to go back to a particular song, consistently

**Comment by rubenwardy** [comment]
* I realised I had party mod on (the music icon doesn't look like a party, it looks like the uploading songs button).
* It then played the same track over and over again when I pressed next. This no longer happened when I changed from do-not-repeat to continuous playback, I guess it had only one song in the queue.
* However, it then seemed to be switching between two albums
* I noticed that two albums where in the list, so I clicked a track then cleared the list
* The player played the track I clicked, then one from the old album, then the rest from the album of the track I picked

Basically, I expected clicking a track in an album to clear the play list and play that track with the rest of the album as coming up

**Comment by gnumdk** [comment]
Juste click on thé play icon on album

**Comment by rubenwardy** [comment]
Thanks! That works well

**Comment by rubenwardy** [comment]
Would you accept a PR to add a setting to preferences > behaviour that determines whether old albums are kept when clicking a track? ie: "Clear albums in playlist on track click"

I've achieved it here without a setting: https://github.com/rubenwardy/lollypop/commit/0506831cfe3fde2090c43c0c95ba3af7a005a53c

I get the impression that you prefer less settings though.

**Comment by gnumdk** [comment]
>I get the impression that you prefer less settings though.
Lollypop is targeting GNOME, not KDE ;)
